docs/updates/November_2023.md

```diff
@@ -0,0 +1,54 @@
+- Slow fundraising allowed me to dive into the code and making a huge dent in wrapping up no oracle work
+- We are following advisement from our lawyers to set up a legal entity in the Cayman Islands
+
+### Asks
+
+- Please get ready to hit that retweet button when we release the documentation and the dApp.
+- We will need to ask for KYC docs as a part of documenting the pre-see raise agreement.
+- Do you know any [Alliance Alumni](https://alliance.xyz/companies) that could put in a good word for us? (we applied)
+
+# Development
+
+## Contracts
+
+Since fundraising conversations have slowed down, I have been able to refocus on wrapping up some of the more technical components of our no oracle work. I had originally looked at the TickMath Library from Uniswap V3 and thought we could refactor it to our needs by changing one line of code. Unfortunately, I was way off. I ended up having to do a bit of a deep dive to fully grok the logic, reviewing auditor write ups like this one from ABDK on [Logarithm-Approximation-Precision](https://hackmd.io/@abdk/SkVJeHK9v#Logarithm-Approximation-Precision). I am on the other side of it now and close to opening a pull request that will give me a much needed tool to proceed with the Tranche work. Here is a rough draft of a [write up of my findings](https://www.notion.so/Generalization-of-Binary-Logarithm-Adjustments-569ae68b2b1b41b8be9f13a72373483b?pvs=21). 
+
+We also introduced the [ERC-4626](https://ethereum.org/en/developers/docs/standards/tokens/erc-4626/) interface that will be important to allow for easy integrations with lending aggregators and also introduces the concept of shares and assets which is crucial for accruing interests. 
+
+## Docs
+
+Last month I got a rough outline of our docs up on our website. This month we added styling, refined the content, and added our contract documentation. Contract documentation allow you to see each contract and all of its functions and events along with the documentation for that function.
+
+## Interface
+
+We want to release the dApp by the end of the year. So we made a priority list and picked the top items we thought we could deliver on. The Pair overview page will get a light touch if not all of the wire frame below. This should help to make if feel a bit more like Aave and Compound and give users quick access to accomplish simple tasks.
+
+![Pool Page](./assets/Pool%20Page.png)
+
+The second goal is to add helper text throughout the app to help give users an easy way to learn more about what things are, what they do, and what they mean. 
+
+The last goal is to rename a few things in the app to try to be more consistent with other daps. One example is that we interchange the words Pool and Pair a lot and it can be confusing. So this is really an exercise in making sure we are using consistent language with apps that we offer feature parity with to help the average DeFi degen feel at home. 
```

Comment:
```suggestion
The last goal is to rename a few things in the dApp to try to be more consistent with other dApps. One example is that we interchange the words Pool and Pair a lot and it can be confusing. So this is really an exercise to help the average degen feel at home and make sure we are using consistent language with dApps with which we offer feature parity. 
```